src/app/@ansyn/ansyn/app-providers/overlay-source-providers/open-aerial-source-provider.ts

```diff
@@ -0,0 +1,106 @@
+import { BaseOverlaySourceProvider, IFetchParams } from 'app/@ansyn/overlays/index';
+import { Observable } from 'rxjs/Observable';
+import { OverlaysOpenAerialFetchData } from '@ansyn/core/models/overlay.model';
+import { StartAndEndDate } from '@ansyn/overlays/models/base-overlay-source-provider.model';
+import { ErrorHandlerService, Overlay } from 'app/@ansyn/core/index';
+import { Inject, Injectable, InjectionToken } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { sortByDateDesc } from "@ansyn/core/utils/sorting";
+import { bboxFromGeoJson, geojsonMultiPolygonToPolygon } from "@ansyn/core/utils/geo";
+import { limitArray } from "@ansyn/core/utils/limited-array";
+import { Response } from "@angular/http";
+import { toRadians } from "@ansyn/core/utils/math";
+import * as wellknown from "wellknown";
+
+const DEFAULT_OVERLAYS_LIMIT = 500;
+export const OpenAerialOverlaySourceType = 'OPEN_AERIAL';
+
+export const OpenAerialOverlaysSourceConfig: InjectionToken<IOpenAerialOverlaySourceConfig> = new InjectionToken('open-aerial-overlays-source-config');
+
+export interface IOpenAerialOverlaySourceConfig {
+	baseUrl: string;
+}
+
+@Injectable()
+export class OpenAerialSourceProvider extends BaseOverlaySourceProvider {
+	sourceType = OpenAerialOverlaySourceType;
+
+	constructor(public errorHandlerService: ErrorHandlerService, protected http: HttpClient, @Inject(OpenAerialOverlaysSourceConfig) protected _overlaySourceConfig: IOpenAerialOverlaySourceConfig) {
+		super();
+
+	}
+
+	fetch(fetchParams: IFetchParams): Observable<OverlaysOpenAerialFetchData> {
+		if (fetchParams.region.type === 'MultiPolygon') {
+			fetchParams.region = geojsonMultiPolygonToPolygon(fetchParams.region as GeoJSON.MultiPolygon);
+		}
+		let bbox = bboxFromGeoJson(fetchParams.region as GeoJSON.Polygon);
+		// if limit not provided by config - set default value
+		fetchParams.limit = fetchParams.limit ? fetchParams.limit : DEFAULT_OVERLAYS_LIMIT;
+		let baseUrl = this._overlaySourceConfig.baseUrl;
+		// add 1 to limit - so we'll know if provider have more then X overlays
+		let urlWithParams = `${baseUrl}?limit=${fetchParams.limit + 1}&bbox=${bbox[0]}%2C${bbox[1]}%2C${bbox[2]}%2C${bbox[3]}`;
+		return this.http.get<OverlaysOpenAerialFetchData>(urlWithParams)
+			.map(data => {
+				return this.extractArrayData(data.results);
+			})
+			.map((overlays: Overlay[]) => limitArray(overlays, fetchParams.limit, {
+				sortFn: sortByDateDesc,
+				uniqueBy: o => o.id
+			}))
+			.catch((error: Response | any) => {
+				return this.errorHandlerService.httpErrorHandle(error);
+			});
+	}
+
+	getById(id: string, sourceType: string): Observable<Overlay> {
+		let baseUrl = this._overlaySourceConfig.baseUrl;
+		let urlWithParams = `${baseUrl}?_id=${id}&properties.sensor=${sourceType}`;
```

Comment:
`http.get` parsing params....
did you check it? (by display overlay and share url)